### PR TITLE
Migrate remaining templates to "Back to top" web component

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/31200

This is the third PR related to the "Back to top" web component migration (#759 , #779). Three templates are modified:

- `checklist`
- `media_list_images`
- `medial_list_videos`

The CMS currently doesn't have content for these, so I used a [special environment for testing](https://dsva.slack.com/archives/CT4GZBM8F/p1638207741315300?thread_ts=1638205734.306100&cid=CT4GZBM8F) provided by the CMS team.

~~**Note:** This PR also inlines the main "Last updated" div into the same container that `<va-back-to-top>` is in to remove some excess margin. This isn't strictly necessary for the migration. If we want to keep that change, we may want to do the same to the templates modified in #759 .~~

^ I'll take care of that in another PR to keep this one more focused.

## Testing done

1. Pull content from the custom Tugboat environment that contains content for the templates:
  `yarn build --pull-drupal --drupal-address https://cms-2jgriyje0hbgvdyy6dtkem57mlcqw2be.demo.cms.va.gov/`

2. Watch the build for any template changes:
  `yarn watch`

3. Inspect in browser :eyes: 

## Screenshots

See review

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/checklist.drupal.liquid [hunk 1]
```diff
@@ -102,12 +102,19 @@
     <!-- Need more help -->
     {% include "src/site/paragraphs/contact_information.drupal.liquid" with entity = fieldContactInformation.entity %}
 
-    <!-- Last updated -->
-    {% include "src/site/includes/last-updated.drupal.liquid" %}
-  </main>
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        <div class="usa-content">
+          <va-back-to-top></va-back-to-top>
```
Comment (bkjohnson) [inline src/site/layouts/checklist.drupal.liquid:108]: ## Before

image


## After

image

File: src/site/components/up_to_top_button.html [hunk 2]
```diff
@@ -1,38 +0,0 @@
...
-=====================
-
-Used as quick navigate back to top for long article pages.
-
-1. To import button, use {% include "src/site/components/up_to_top_button.html" %}.
-2. It needs to be imported as the last component within the div with the "usa-content" class.
-
-{% endcomment %}
-
-<div id="back-to-top-container">
```
Comment (bkjohnson) [inline src/site/components/up_to_top_button.html:14]: This file is no longer imported :tada:

**Review by bkjohnson — COMMENTED** [review]
I wanted to call out the changes around "Last updated". Let me know if I should leave them out of this PR, or handle the margin change differently.

File: src/site/layouts/media_list_videos.drupal.liquid [hunk 3]
```diff
@@ -105,12 +105,17 @@
     <!-- Need more help -->
     {% include "src/site/paragraphs/contact_information.drupal.liquid" with entity = fieldContactInformation.entity %}
 
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        <div class="usa-content">
+          <va-back-to-top></va-back-to-top>
```
Comment (bkjohnson) [inline src/site/layouts/media_list_videos.drupal.liquid:111]: ## Before

image


## After

image

**Review by kelsonic — APPROVED** [review]
looks great! 🎉 thank you for adding before/after screenshots again! 🚀

**Pull request merged**